spec/lib/sysloggable/logger_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "spec_helper"
+
+describe Sysloggable::Logger do
+  let(:syslogger) { spy("syslogger") }
+  subject(:logger) { described_class.new(ident: "test_ident") }
+
+  before do
+    Sysloggable::Container.stub('lib.syslogger', syslogger)
+  end
+
+  described_class::SEVERITIES.each do |severity_name, severity_value|
+    it "responds to #{severity_name}" do
+      sev_label = (severity_name == :unknown) ? "ANY" : severity_name.upcase
+
+      expect(syslogger).to receive(:add).
+        with(severity_value, "severity=#{sev_label} service=test_ident operation= duration=0 message=msg")
+
+      logger.send(severity_name, "msg")
+    end
+  end
+
+  it "counts duration" do
+    Timecop.freeze(Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.